frontend/package.json

```diff
@@ -74,84 +74,13 @@
   },
   "devDependencies": {
     "env-cmd": "^10.1.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-import-resolver-alias": "^1.1.2",
-    "eslint-plugin-prettier": "^4.2.1",
-    "prettier": "2.8.7",
-    "watch": "^1.0.2"
+    "prettier": "^2.6.1"
   },
   "eslintConfig": {
-    "plugins": [
-      "prettier",
-      "import"
-    ],
     "extends": [
       "react-app",
-      "react-app/jest",
-      "plugin:prettier/recommended",
-      "plugin:import/recommended"
-    ],
-    "settings": {
-      "import/resolver": {
```

Comment:
This might cause all absolute imports to fail

Reply:
It shouldn't. `settings` are declared for `eslintConfig`, so the only thing that can fail is linting. It worked fine in pipeline. I assume, that it's because we returned back to original linting in this PR